SIMULATION_SCHEMA_CRITICAL_ISSUES.md

```
@@ -0,0 +1,453 @@
+
+**Severity:** CRITICAL
+**Impact:** Application will fail to perform JOINs, causing N+1 query problems
+
+### Problem
+
+None of the 8 Sequelize models define associations. The models only define fields but no relationships.
+
+### Evidence
+
+**File:** /home/omar/claude - sales_auto_skill/sales-automation-api/src/models/SimulationRun.js
+```javascript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)